staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -357,6 +357,13 @@ message PodSandboxConfig {
     map<string, string> annotations = 7;
     // Optional configurations specific to Linux hosts.
     LinuxPodSandboxConfig linux = 8;
+    // termination_grace_period is the time that a pod reaper should wait
+    // before terminating the containers in a pod.
+    // For e.g., on a Linux system, this can be used to configure the
+    // systemd TimeoutStopUSec property for the containers running in systemd
+    // scopes, so systemd gives the pod containers enough time to shutdown
+    // gracefully on system shutdown or reboot.
+    int64 termination_grace_period = 9;
```

Comment:
@jterry75 brings up a good point - why is this called `termination_grace_period` if it's for shutdown?

Reply:
@egernst - does this make sense for Kata?

Reply:
Meant to align with the Pod terminationGracePeriodSeconds field

Reply:
This is for graceful termination at either shutdown or reboot, so I feel that `termination_grace_period` covers both of those cases in the name.